chart/templates/webserver/webserver-ingress.yaml

```
@@ -39,31 +39,38 @@ spec:
   {{- if .Values.ingress.web.tls.enabled }}
   tls:
     - hosts:
-        - {{ .Values.ingress.web.host }}
+{{- if .Values.ingress.web.tls.enabled }}
+        {{- range .Values.ingress.web.hosts | default (list .Values.ingress.web.host) }}
+        - {{ . | quote }}
+        {{- end }}
```

```suggestion
        {{- .Values.ingress.web.hosts | default (list .Values.ingress.web.host) | toYaml | nindent 8 }}
```

I think using `toYaml` is cleaner, thoughts?

---

Yup. I was being silly, people will have to quote their own strings when passing them in if they have special characters.

---

Actually, I am not seeing this bear out in my testing.  Some hostnames that need quotes are remaining unquoted with the above change.

---

Ignore me again. All good here.